tests/fake_beamline_all_devices_rasie_exception.py

```diff
@@ -0,0 +1,21 @@
+from bluesky.protocols import Readable
+from ophyd import EpicsMotor
+from ophyd.utils import (
+    DestroyedError,
+    DisconnectedError,
+    UnknownStatusFailure,
+)
+
+from dodal.devices.cryostream import Cryo
+
+
+def device_a() -> Readable:
+    raise DestroyedError
+
+
+def device_b() -> EpicsMotor:
+    raise DisconnectedError
+
+
+def device_c() -> Cryo:
+    raise UnknownStatusFailure
```

Comment:
Currently one Protocol, 2 ophyd devices, add an ophyd-async device class?

Reply:
Given that we are moving in the direction of exclusive ophyd-async (and therefore hopefully dodal's handling of ophyd devices can go away one day) I might just make it ophyd-async exclusively

Reply:
I'd prefer to see both tested until there are no ophyd devices, given they are different execution paths

Reply:
Have reverted cryostream